### DISCUSSION TITLE
[Kavita] 3rd mouse button (Scroll Wheel) to open in new tab

### DISCUSSION THREAD

**Discussion by DieselTech** [Kareadita/Kavita#2549] [Ideas]
### Idea Description

Posted by Spectrem
· Aug 2022
· 17 Upvotes
https://feats.kavitareader.com/posts/196/3rd-mouse-button-scroll-wheel-to-open-in-new-tab

Description

If you Press the 3rd mouse button (Scroll Wheel) you should be able to open a series in a new tab.

### Duration of Using Kavita

42

### Idea Category

Feature Enhancement

### Agreement

- [X] I agree that this is solely for submitting ideas, and I will search for existing ideas before posting.

**Comment by Tropaion** [comment]
Any news on this? This is the feature I miss the most from kavita reader.

**Reply by DieselTech** [reply]
When it was last looked into it was more complex than initially thought due to how the virtual display parts are rendered on screen.

**Comment by blumarten** [comment]
i just started using kavita as a catalogue for a bunch of PDFs and would love a way of having a PDF open in a new chrome tab when clicking it. Once i've read a doc and go back it's a bit of a drag to try and find where I was up to in the main series view as currently i'm deposited back at the top of a big list of very similarly-named files.

for reference, i use a macbook with a touch pad, so generally open links in new tabs with cmd+click, or right-click > open in new tab, so no third mouse wheel for me here.

**Comment by majora2007** [comment]
This can now be closed. With the UX rework, almost all parts of Kavita are now handled that way.
